lib/internal/process/pre_execution.js

```diff
@@ -274,16 +277,13 @@ function setupWebCrypto() {
   }
 
   if (internalBinding('config').hasOpenSSL) {
-    const webcrypto = require('internal/crypto/webcrypto');
-    ObjectDefineProperty(globalThis, 'crypto',
-                         { __proto__: null, ...ObjectGetOwnPropertyDescriptor({
-                           get crypto() {
-                             return webcrypto.crypto;
-                           }
-                         }, 'crypto') });
-    exposeInterface(globalThis, 'Crypto', webcrypto.Crypto);
-    exposeInterface(globalThis, 'CryptoKey', webcrypto.CryptoKey);
-    exposeInterface(globalThis, 'SubtleCrypto', webcrypto.SubtleCrypto);
+    defineReplaceableLazyAttribute(
+      globalThis, 'internal/crypto/webcrypto', ['crypto'], false
+    );
```

Comment:
@aduh95 @joyeecheung Do you have a suggestion on how to add the following to the lazy loaded `crypto` getter now?

```
if (this !== globalThis && this != null)
  throw new ERR_INVALID_THIS(
    'nullish or must be the global object');
```

Reply:
`defineReplaceableLazyAttribute()` is just a helper that for code reuse in similar use cases, if the use cases stop being similar, then it's better to just write something tailored to your needs (e.g. you can copy the code of `defineReplaceableLazyAttribute()` and customize the getter) .

Reply:
that would only really work for the first invocation tho, i'll ponder on this some more.

Reply:
[Got it](https://github.com/panva/node/compare/20cd51db532b123b12941e79c8086e443255ed4d...bade49c3bf3529fb00f58868c35aafc0e86c638f). Once the existing [PRs](https://github.com/nodejs/node/pulls/panva) touching the WPTs get resolved I'll open this up.